CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+### 3.10.0 (2017-3-20)
```

Comment:
The changelog should get updated outside of a PR, after all changes have be merged to master. And it can be run on its own without a review.